packages/ui/src/composites/wui-otp/index.ts

```diff
@@ -17,6 +17,8 @@ export class WuiOtp extends LitElement {
 
   private numerics: WuiInputNumeric[] = []
 
+  private valueArr: string[] = [...Array(this.length)].map(() => '')
```

Comment:
You can also use `Array.from` for this use case, it makes it a bit more readable
```suggestion
  private valueArr: string[] = Array.from({ length: this.length }, () => '')
```